libs/angular-google-charts/src/lib/components/chart-wrapper/chart-wrapper.component.ts

```diff
@@ -66,7 +66,7 @@ export class ChartWrapperComponent implements ChartBase, OnChanges, OnInit {
     return this.wrapper;
   }
 
-  public set chartWrapper(wrapper: google.visualization.ChartWrapper) {
+  public set chartWrapper(wrapper: google.visualization.ChartBase | undefined) {
```

Comment:
This doesn't compile when using TS strict mode. Can you explain the reasoning behind this change?